lib/Table/TableColumn.ts

```diff
@@ -252,6 +252,30 @@ export default class TableColumn {
       }
     };
 
+    let yyyyQQ: StringToDateFunction = value => {
+      // Is it quarterly data in the format yyyy-Qx ? (Ignoring null values, and failing on any purely numeric values)
+      if (value.indexOf("-Q") === 4) {
```

Comment:
I think this will be a tad more faster when parsing many rows if we do something like `value.slice(4,6) === "-Q"`.

Reply:
yep good idea!